annotation-file-utilities/src/annotator/find/Insertion.java

```diff
@@ -203,6 +247,50 @@ public static void setAlwaysQualify(Set<String> set) {
     }
 
     /**
+     * Gets the type.  It is assumed that the returned value will be
```

Comment:
Avoid passive voice.  Do you mean, "The client must modify the returned value to update the type to be inserted."?
How does the client do this, and why?  Is it always required or only sometimes?

Also, rather than just "the type", I think it would be clearer to say "the type to be inserted, if any".  Similarly clarify `setType` below.